.github/workflows/phpunit.yml

```diff
@@ -34,12 +34,14 @@ jobs:
       fail-fast: false
       matrix:
         php-versions: ['8.0', '8.1', '8.2', '8.3']
-        server-versions: ['stable28', 'stable29', 'stable30', 'master']
+        server-versions: ['stable28', 'stable29', 'stable30', 'stable31', 'master']
         exclude:
           - php-versions: '8.0'
             server-versions: 'master'
           - php-versions: '8.0'
             server-versions: 'stable30'
+          - php-versions: '8.0'
+            server-versions: 'stable31'
```

Comment:
Nc 30 drops support for php 8.0 and it's end of life by now: https://github.com/nextcloud/server/wiki/Releases-and-PHP-versions

Shall we just drop support for nc <30 in this pr ?